lib/db_utils.go

```diff
@@ -479,6 +512,9 @@ func StatePrefixToDeSoEncoder(prefix []byte) (_isEncoder bool, _encoder DeSoEnco
 	} else if bytes.Equal(prefix, Prefixes.PrefixDAOCoinLimitOrderByOrderID) {
 		// prefix_id:"[62]"
 		return true, &DAOCoinLimitOrderEntry{}
+	} else if bytes.Equal(prefix, Prefixes.PrefixGroupMembershipIndex) {
```

Comment:
We should also add prefixes `64`, and `65` with `return false, nil`. You can check a similar situation with prefix `PrefixPublicKeyBlockHashToBlockReward` and others